realtime/src/main/java/com/mx/path/api/lib/realtime/AccountCreateRequest.java

```diff
@@ -8,9 +8,13 @@
 import com.mx.common.accessors.UnauthorizedException;
 import com.mx.common.http.HttpStatus;
 import com.mx.common.http.MediaType;
-import com.mx.path.api.lib.realtime.models.MdxAccount;
 import com.mx.path.gateway.net.Request;
+import com.mx.path.service.connection.realtime.model.MdxAccount;
 
+/**
+ * @deprecated Use com.mx.path.service.connection.realtime
```

Comment:
Not a huge deal, but this can be turned into a link.

Like this:

```java
{@link com.mx.path.service.connection.realtime.MdxRealtimeConnection}
```

Reply:
Nice! Done.